src/lib.rs

```diff
@@ -223,10 +223,11 @@ pub enum EaseMethod {
 }
 
 impl EaseMethod {
+    #[must_use]
     fn sample(self, x: f32) -> f32 {
         match self {
             EaseMethod::EaseFunction(function) => x.calc(function),
-            EaseMethod::Linear => x,
+            Self::Linear => x,
```

Comment:
This is a bit awkward, why change only that one and not the ones above and below? 😅

Reply:
Good question, I guess that's a clippy bug. Fixed.